### DISCUSSION TITLE
Runway fields on Statamic entities and vice versa

### DISCUSSION THREAD

**Discussion by janhenkes** [statamic-rad-pack/runway#604] [Help]
### Description

I am a bit stuck on how to link a User (stored in the database) to an Organization and vice versa. I tried creating BelongsTo and HasMany fields but that does not seem to work. Is this something runway already supports? 
Or do you have a hint on how I could build this myself (for example with a custom field type)?

### Environment

Environment
Application Name: ESLHO EQA Portal
Laravel Version: 10.48.22
PHP Version: 8.2.22
Composer Version: 2.3.5
Environment: local
Debug Mode: ENABLED
URL: eslho-eqa-portal.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: file
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 1
Sites: 1
Stache Watcher: Enabled
Static Caching: Disabled
Version: 5.25.0 PRO

Statamic Addons
statamic-rad-pack/runway: 7.9.3

**Comment by janhenkes** [comment]
@duncanmcclean I have done some more research on this. Is the only currently supported way to achieve this to create a custom user profile screen using runway?

**Reply by duncanmcclean** [reply]
Apologies, I must have missed the notification for this previously.

Yes, if you want to display a user's related models, you'll need to add the `User` model as a Runway resource and have a separate "profile" page for the user.
